index.js

```diff
@@ -37,12 +37,20 @@ function record(voiceConnection) {
     voiceDataStream.on("error", console.error);
 }
 
+function collectionValuesContain(c, o) {
+    for (let [k, v] of c) {
+        if (v == o) return true;
+    }
+    return false;
+}
+
 function tts(message) {
     if (getMe(message.channel.guild).voiceState == null) {
         playDict.splice(playDict.indexOf(message.channel));
         return;
     }
     voiceConnection = client.voiceConnections.get(message.channel.guild.id);
+    if (!collectionValuesContain(voiceConnection.channel.members, message.author.fetchMember()) return;
```

Comment:
```suggestion
    if (!message.channel.guild.channels.get(getMe(message.channel.guild).voiceState.channelID).members.has(message.author.id)) return;
```
This is a _better_ solution to only including connected members.

Reply:
*such better*

Reply:
Well, it is better in the sense that it will work.